app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsViewModel.kt

```diff
@@ -152,7 +152,7 @@ class NotificationsViewModel @Inject constructor(
 
     private fun shouldFilterStatus(notificationViewData: NotificationViewData): Filter.Action {
         return when ((notificationViewData as? NotificationViewData.Concrete)?.type) {
-            Notification.Type.MENTION, Notification.Type.STATUS, Notification.Type.POLL -> {
```

Comment:
Contrary to their name, `Notification.Type.STATUS` can't be filtered (they are always notifications about the users own posts)